test/tracing/test_tracer.py

```diff
@@ -1,5 +1,16 @@
+import builtins
+import sys
 from unittest.mock import Mock
 
+import opentelemetry.trace
+import pytest
+from _pytest.monkeypatch import MonkeyPatch
```

Comment:
why not using the fixture? (i.e. just taking a `monkeypatch` param in the test function)

Reply:
I'm doing that - this is just the typing